kafka_utils/util/offsets.py

```diff
@@ -546,7 +546,7 @@ def set_consumer_offsets(
 def _nullify_partition_offsets(partition_offsets):
     result = {}
     for partition in partition_offsets:
-        result[partition] = -1
+        result[partition] = 0
```

Comment:
@fede1024 I ran some tests, before and after this change. 

When the value was set to -1, this was the output which I was getting pb - 253964
When I set it to 0, the output I am getting this is pb - 253967

I have added acceptance tests to test the same. I got this value purely by trial and error. I wasn't able to find actual acceptance tests which were testing this before.
@ecanzonieri I am trying to locate the kafka code, and I will post it once I locate it.